module/data/actor/templates/sentient.mjs

```diff
@@ -187,11 +187,16 @@ export default class SentientTemplate extends CommonTemplate {
                         required: true,
                         nullable: false,
                         min: 0,
-                        step: 1,
+                        // step: 1,
                         initial: 0,
                         integer: true,
                         label: "ED.Actor.Characteristics.recoveryTestsCurrent"
                     } ),
+                    stun: new fields.BooleanField( {
```

Comment:
Does that mean, that you're attacking for stunning? If so, maybe take a more descriptive name.
Also, shouldn't that be part of condition, like e.g. aggressive attack?

Reply:
no, it does not, there is a special rule for recovering from stun damage, but you can do that only once after receiving stun damage. I had the idea to change this boolean after each recovery check. One hour has to be between each recovery checks, so I thought that would be okay.

stun recovery is adding will powe step to toughness step but regenerates only stun damage

Reply:
Bessere Bezeichnung

Reply:
renamed in stunRecoveryAvailability